### ISSUE TITLE
WinUI3 preview 3 - Shadow animation of opened flyouts is shown twice

### ISSUE THREAD

**Issue by lhak** [microsoft/microsoft-ui-xaml#6206]
### Describe the bug

When a MenuFlyout is opened, the shadow animation is shown twice

### Steps to reproduce the bug

1. Create a new application
2. Add the following XAML:

```xaml
<Grid  ContextFlyout="{x:Bind TestFlyout}">
  <Grid.Resources>
    <MenuFlyout x:Name="TestFlyout" >
      <MenuFlyoutItem Text="Test"/>
    </MenuFlyout>
  </Grid.Resources>
</Grid>
```

Open the context menu

### Expected behavior

Animation should only be shown once

### Windows app type

- [ ] UWP
- [X] Win32

### Device form factor

Desktop

### Windows version

Insider Build (xxxxx)

**Comment by seven-mile** [comment]
The shadow animation of most user flyouts indeed flicker. (Is it losing focus?)

![](https://p.sda1.dev/3/70bcb7451d0ee4262b8f034d4d270861/2021-11-03-18-52-42.gif)

But the menu flyout of CommandBar seems to work correctly. Why?

![](https://p.sda1.dev/3/aabb1d4e1092b6e90fd2a0c4b1605c51/2021-11-03-18-52-13.gif)

(If the GIF stopped, open it in a new tab) FYI.

**Comment by StephenLPeters** [comment]
@codendone FYI

**Comment by bogdan-patraucean** [comment]
I'm having the same problem, it's only happening in WASDK apps.

**Comment by lhak** [comment]
This is still an issue in the latest 1.3 release. There is a short flicker before the shadow animation is played.

**Comment by ranjeshj** [comment]
This looks fixed in WASDK 1.4 !
